CMakeLists.txt

```diff
@@ -337,6 +337,26 @@ endif (WIN32)
 
 endif (OVERRIDE_CMAKE_CXX_FLAGS)
 
+OPTION(CMAKE_ENABLE_CODE_COVERAGE "Enable code coverage" OFF)
+
+if(CMAKE_ENABLE_CODE_COVERAGE)
+
+  string(TOUPPER ${CMAKE_BUILD_TYPE} UPPERCASE_CMAKE_BUILD_TYPE)
+  if(NOT UPPERCASE_CMAKE_BUILD_TYPE STREQUAL DEBUG)
+    message( WARNING "Code coverage results with an optimised (non-Debug) build may be misleading" )
+  endif(NOT UPPERCASE_CMAKE_BUILD_TYPE STREQUAL DEBUG)
+
+  if(CLR_CMAKE_PLATFORM_UNIX)
+    add_definitions(-fprofile-arcs -ftest-coverage -fPIC)
```

Comment:
Ok, thanks. So the compilation of the libcorguids.a with code coverage adds a dependency that makes it necessary to be built with the -fPIC. Could you please try to add `add_compile_options(-fPIC)` to `src\inc\CMakeLists.txt` before the `endif(WIN32)`? It would be nice to fix the single place where we were missing the option instead of adding it globally.